machine_learning/Random Forest/RandomForest.py

```diff
@@ -0,0 +1,520 @@
+"""
+@author: Manish Kumar Chintha
+"""
+
+"""
+list of libraries imported for the program to work.
+"""
+import math
+import numpy as np
+import csv
+from collections import defaultdict
+import operator
+import random
+
+"""
+Class for DecisionTree classifier and all of it's related model.
+I have called this class in RandomForest Class to build multiple tree models
+and saved the objects and models in RandomForest class to use for classification
+
+Building a Decision Tree and testing it on  training data.
+I've separated the Decision Tree classifier and related function,
+so that I can even test it independently.
+"""
+
+
+class DecisionTree(object):
```

Comment:
Please make this a https://docs.python.org/3/library/dataclasses.html
```suggestion
@dataclass
class DecisionTree:
```